database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE zn_data.storage (
```

Comment:
It's important to switch roles before running this migration. In order not to forget about it, I suggest to add this line first:

`SET ROLE zalando_nakadi_data_owner;`

Otherwise, tables are created with wrong permissions and Nakadi is unable to access it.